pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -245,6 +250,14 @@ func (b *Build) BuildPod(images BuildPodImages, buildContext BuildContext) (*cor
 		Resources: b.Spec.Resources,
 		Args: args(
 			[]string{"-layers=/layers", "-analyzed=/layers/analyzed.toml"},
+			func() []string {
+				if !platformAPILessThan07 {
+					if b.Spec.RunImage.Image != "" {
```

Comment:
I think the variable can be used here and elsewhere (`runImage`).

Regardless of where our source of truth `runImage` came from we can still pass it through in all these places (even though we didn't before)

Reply:
@tylerphelan 
you mean to change it to:
```golang
if !platformAPILessThan07 {
   return []string{"-run-image=" + runImage}
}
```
rigth?

Reply:
right

Reply:
and `if b.Spec.RunImage.Image != ""`

Reply:
Fixed on [commit](https://github.com/pivotal/kpack/pull/988/commits/c190e3f3e8744b25ba10210409e1ce3ca58013c0)